packages/react-docs/example/public/css/roboto.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: 'Roboto';
+  src: url('../fonts/Roboto-Thin.ttf') format('truetype');
+  font-weight: 100;
+  font-style: normal;
+}
+
+/*@font-face {
```

Comment:
a few font faces commented out. Remove these?

Reply:
These are here because we might want to include them at a later date. Added a comment about this exception to the `no commented out code` policy.